addons/website_sale/static/src/js/product/product.js

```diff
@@ -0,0 +1,26 @@
+/** @odoo-module **/
+
+import { formatCurrency } from '@web/core/currency';
+import { patch } from '@web/core/utils/patch';
+import { Product } from '@sale/js/product/product';
+
+patch(Product, {
+    props: {
+        ...Product.props,
+        strikethrough_price: { type: Number, optional: true },
+        can_be_sold: { type: Boolean, optional: true },
+        category_name: { type: String, optional: true },
+        currency_name: { type: String, optional: true },
```

Comment:
nitpicking: Don't hesitate to add comments for weird params
```suggestion
        currency_name: { type: String, optional: true },  // For tracking
```